parse-deploy-file.yaml

```diff
@@ -49,24 +49,19 @@ run:
       ip_ranges = [lookup['DOM1']]
 
     auth_required = not data.get('disable_authentication', False)
+    webapp_port = data.get('port', 80)
 
+    # The following are input values of the webapp helm chart
     with open('deploy-params/overrides.yaml', 'w') as output:
       json.dump({
         'AuthProxy': {
           'AuthenticationRequired': auth_required,
-          'IPRanges': ','.join(ip_ranges)
-        }
+          'IPRanges': ','.join(ip_ranges),
+        },
+        'WebApp': {
+          'Port': webapp_port,
+        },
       }, output)
     print('Wrote deploy-params/overrides.yaml')
 
-    os.mkdir('deploy-params/auth-proxy')
```

Comment:
This is not used by the `webapp_pipeline`, it was meant to make this task more generic but I'm simplifying it.

The [`webapp_pipeline`](https://github.com/ministryofjustice/analytics-platform-concourse-github-org-resource/blob/d58555fcfdbd9e62cb03b9af0dd5b7e04776637e/resource/webapp_pipeline.yaml#L53) is only using the `deploy-params/overrides.yaml` file.